web/templates/pages/add_access_key.html

```diff
@@ -0,0 +1,81 @@
+            <a class="ui-button cancel" dir="ltr" id="btn-back" href="/list/access-key/"><i
+                        class="fas fa-arrow-left status-icon blue"></i><?= _('Back'); ?></a>
+        </div>
+        <div class="l-unit-toolbar__buttonstrip float-right">
+            <a href="#" class="ui-button" data-action="submit" data-id="vstobjects"><i
+                        class="fas fa-save status-icon purple"></i><?= _('Save'); ?></a>
+        </div>
+    </div>
+</div>
+<!-- End toolbar -->
+
+<div class="l-separator"></div>
+
+<div class="l-center animated fadeIn">
+
+    <form id="vstobjects" name="v_add_access_key" method="post">
+        <input type="hidden" name="token" value="<?= $_SESSION['token'] ?>"/>
+        <input type="hidden" name="ok" value="Add"/>
+
+        <table class="data mode-add">
+            <tr class="data-add">
+                <td class="data-dotted">
+                    <table class="data-col1">
+                        <tr>
+                            <td></td>
+                        </tr>
+                    </table>
+                </td>
+                <td class="data-dotted">
+                    <table class="data-col2">
+                        <tr>
+                            <td class="step-top">
+                                <span class="page-title"><?= _('Adding Access Key'); ?></span>
+                            </td>
+                        </tr>
+                        <tr>
+                            <td>
+                                <?php show_error_panel($_SESSION); ?>
+                            </td>
+                        </tr>
+
+                        <tr>
+                            <td class="vst-text input-label">
+                                <?= _('Permissions'); ?>
+                            </td>
+                        </tr>
+                        <?php
+                        foreach ($apis as $api_name => $api_data) {
+                            ?>
+                            <tr id="send-welcome">
+                                <td class="vst-text input-label">
+                                    <label><input type="checkbox" size="20" class="vst-checkbox"
+                                                  name="v_apis[]" value="<?= $api_name ?>"
+                                                  tabindex="5"/><?= _($api_name); ?></label>
+                                </td>
+                            </tr>
+                            <?php
+                        } ?>
+
+                        <tr>
+                            <td class="vst-text step-top">
+                                <?= _('Comment'); ?> <span class="optional">(<?= _('optional'); ?>)</span>
+                            </td>
+                        </tr>
+                        <tr>
+                            <td>
+                                <input type="text" size="20" class="vst-input" name="v_comment" maxlength="255"
```

Comment:
2022/04/18 19:08:03 [error] 1012285#0: *30 FastCGI sent in stderr: "PHP message: PHP Warning:  Undefined variable $v_comment in /usr/local/hestia/web/templates/pages/add_access_key.html on line 72PHP message: PHP Deprecated:  trim(): Passing null to parameter #1 ($string) of type string is deprecated in /usr/local/hestia/web/templates/pages/add_access_key.html on line 72" while reading response header from upstream,